test/test_core.py

```diff
@@ -7454,7 +7454,7 @@ def test_demangle_stacks_symbol_map(self):
     self.assertIsNotNone(full_aborter)
     self.assertIsNotNone(short_aborter)
     print('full:', full_aborter, 'short:', short_aborter)
-    if config.SPIDERMONKEY_ENGINE and os.path.exists(config.SPIDERMONKEY_ENGINE[0]):
+    if config.SPIDERMONKEY_ENGINE and config.SPIDERMONKEY_ENGINE == self.js_engines[0]:
```

Comment:
Why check if it's the first item in `js_engines`? Shouldn't we run the test if it is the second in the list?

Reply:
I guess we could... I was thinking about that case when no `SPIDERMONKEY_ENGINE` at all was configured..  but I guess having `None` as  `self.js_engines[0]` would already be in undefined behaviour territory?